src/services/savedResearches.ts

```diff
@@ -132,6 +132,7 @@ export const fetchLastCohorts = async (): Promise<FormattedCohort[] | undefined>
             favorite: cohort.favorite
           }))
           .filter(Boolean)
+          .reverse()
```

Comment:
Doesn't the ordering param in the query to cohort back work?

Reply:
weird, it shoud work, I'll take a look at the backend

Reply:
fixed